docs/openapi-docs/src/main/scala/sttp/tapir/docs/openapi/OpenAPIDocsOptions.scala

```diff
@@ -3,11 +3,13 @@ package sttp.tapir.docs.openapi
 import sttp.model.Method
 import sttp.tapir.SchemaType.SObjectInfo
 import sttp.tapir.docs.apispec.defaultSchemaName
+import sttp.tapir.{EndpointOutput, stringBody}
 
 case class OpenAPIDocsOptions(
     operationIdGenerator: (Vector[String], Method) => String,
     schemaName: SObjectInfo => String = defaultSchemaName,
-    referenceEnums: SObjectInfo => Boolean = _ => false
+    referenceEnums: SObjectInfo => Boolean = _ => false,
+    default400Output: EndpointOutput[_] = stringBody.description("Input validation failed.")
```

Comment:
I think I'd still make this optional. Maybe `defaultDecodeFailureOutput` would be a better name. Or maybe we could move the logic here, having a `EndpointInput => Option[ValuedEndpointOuput[_]]` function in the options. The default implementation would contain partially what's now in `inputToDefaultErrorResponses`. WDYT?

The logic would be nicely pluggable then, providing a way to implement the decision logic in an app-specific way. But again, maybe that's not a place where you need the flexibility?

Reply:
Yeah, I'll make this optional in case users don't like or need the functionality. I think this could be an `EndpointInput[_] => Option[EndpointOutput[_]]`, potentially implemented using `DefaultDecodeFailureHandler.FailureMessages.failureSourceMessage`. And yes, I think I'll move some logic from `inputToDefaultErrorResponses`.

(I think we don't need to return `ValuedEndpointOutput`, because we don't need the value returned by the endpoint to generate the docs, is that correct?)